cybox/common/vocabs.py

```diff
@@ -6,13 +6,58 @@
 from cybox.common import PatternFieldGroup
 from cybox.utils import normalize_to_xml, denormalize_from_xml
 
-INVALID_XSI_TYPE = "bad_xsi_type"
+
+class VocabField(cybox.TypedField):
```

Comment:
Also, would this be better defined in `common` (next to TypedField rather than next to VocabString)? It cleans up some of the imports above, and is likely to get moved to mixbox soon.